### DISCUSSION TITLE
This channel is keep repeating the same content

### DISCUSSION THREAD

**Discussion by iotv0815** [iptv-org/community#969] [Show and tell]
"ZhejiangSatelliteTVInternational.cn"
"Zhejiang Satellite TV International"
URL: "https://hw-m-l.cztv.com/channels/lantian/channel10/1080p.m3u8"

the response remains the same, as below: it looks frozen at 2023-09-11

#EXTM3U
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:287140087
#EXT-X-TARGETDURATION:8
#EXT-X-PROGRAM-DATE-TIME:2023-09-11T12:59:18+0800
#EXTINF:8.00
1080p/0/00008ab1eaa9fc63_389124_fea50.ts
#EXTINF:8.00
1080p/0/00008ab1eaa9fc63_487b74_113584.ts
#EXTINF:8.00
1080p/0/00008ab1eaa9fc63_59b0f8_173980.ts

above channel is repeating the same content for more than a week.
please notify the right person to recover it.
or if you know someone to contact please let me know.

thanks

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
